### PR TITLE
fixes building energy balance plots

### PR DESCRIPTION
**Pull request by gabriel-happle**
- change sign of `I_rad` (line 170 and 171)
- plus some pep8 in the file

Testing by running the energy balance plot and checking if I_rad is < 0 
(previously it was >0)

### CODE REVIEW

File: cea/plots/demand/energy_balance.py [hunk 1]
```diff
@@ -159,25 +158,25 @@ def calc_graph(analysis_fields, data_frame):
...
     """
     calculates heat flux balance for buildings on hourly basis
 
-    :param data_frame:
+    :param data_frame: demand information of building in pd.DataFrame
+    :param normalize_value: value for normalization of thermal energy fluxes, usually GFA
     :return:
     """
 
+    # invert sign of I_rad, I_rad was previously negative but it was changed to be consistent with ISO 13790
```
Comment (daren-thomas) [inline cea/plots/demand/energy_balance.py]: the comment "I_rad was previously negative but it was changed" is hard to understand without a historical context of the code. Is there a better way to state this?

What is I_rad like before changing signs? Why do we change signs?

File: cea/plots/demand/energy_balance.py [hunk 2]
```diff
@@ -159,25 +158,25 @@ def calc_graph(analysis_fields, data_frame):
...
+
     # calculate losses of heating and cooling system in data frame and adjust signs
     data_frame['Qhs_loss_sen_kWh'] = -abs(data_frame['Qhs_em_ls_kWh'] + data_frame['Qhs_dis_ls_kWh'])
     data_frame['Qhs_tot_sen_kWh'] = data_frame['Qhs_sen_sys_kWh'] + abs(data_frame['Qhs_loss_sen_kWh'])
     data_frame['Qcs_loss_sen_kWh'] = -data_frame['Qcs_em_ls_kWh'] - data_frame['Qcs_dis_ls_kWh']
     data_frame['Qcs_tot_sen_kWh'] = data_frame['Qcs_sen_sys_kWh'] - abs(data_frame['Qcs_loss_sen_kWh'])
-    data_frame['Qcs_tot_lat_kWh'] = -data_frame['Qcs_lat_sys_kWh']  # change sign because this variable is now positive in demand
-
-
+    # change sign because this variable is now positive in demand
```
Comment (daren-thomas) [inline cea/plots/demand/energy_balance.py]: here again "is now positive in demand" refers to historical context of the code (i'm guessing) and i think that adds a dimension we shouldn't be adding. a comment describing _why_ the value needs to be changed (just for the plot, right?) would be interesting. (also, the totals is just the system stuff?)

File: cea/plots/demand/energy_balance.py [hunk 3]
```diff
@@ -253,13 +253,10 @@ def calc_monthly_energy_balance(data_frame, normalize_value):
...
 def main():
     import cea.config
-    import cea.inputlocator
     import cea.plots.cache
     config = cea.config.Configuration()
-    locator = cea.inputlocator.InputLocator(config.scenario)
-    # cache = cea.plots.cache.PlotCache(config.project)
     cache = cea.plots.cache.NullPlotCache()
-    EnergyBalancePlot(config.project, {'building': locator.get_zone_building_names()[0],
+    EnergyBalancePlot(config.project, {'building': config.plots.building,
```
Comment (martin-mosteiro) [inline cea/plots/demand/energy_balance.py]: @gabriel-happle This was actually me, not @daren-thomas (his user was signed in on the computer I made these edits from).

Changing this line allows you to run energy_balance.py with a selected building from the config file. Before, you could either run all plots for a building specified in the config file (even if you didn't need/want the other plots) or you could run the energy_balance plot for whatever the first building in the zone shapefile is. I think this change adds useful functionality.

**Comment by gabriel-happle** [comment]
@daren-thomas and @martin-mosteiro I updated the comments.

**Pull request merged**